ttnn/cpp/ttnn/operations/sliding_window/sliding_window.hpp

```diff
@@ -43,19 +43,21 @@ struct SlidingWindowConfig {
     uint32_pair_t pad_hw_;
     uint32_pair_t dilation_hw_;
 
+    uint32_t groups;
+
     // parallel configuration
     bool has_parallel_config_;
```

Comment:
could you default initialize all parameters too because you have constructor which takes nothing and initialies core_range_set only?

Reply:
@sankarmanoj-tt see
https://github.com/tenstorrent/tt-metal/blob/main/best_practices.md#15-initialize-primitive-types-on-declaration